### ISSUE TITLE
Installing phantomjs on a PC

### ISSUE THREAD

**Issue by dominikwilkowski** [WestpacCXTeam/GUI-source#286]
When installing the blend dependencies PC will throw errors when running grunticon and it's dependency `phantomjs`.

# TL;DR

Replace the `phantomjs.exe` in `node-modules\sbg\node_modules\grunt-grunticon\node_modules\grunticon-lib\node_modules\svg-to-png\node_modules\phantomjs-prebuilt\lib\phantom\bin` with https://storage.googleapis.com/google-code-archive-downloads/v2/code.google.com/phantomjs/phantomjs-1.9.2-windows.zip

# More details

Running "grunticon:STG" (grunticon) task:

```shell
PANTOM ERROR: Error: Error: fail: Phantom had an error opening - C:\Users\L080777\AppData\Local\Temp\grunticon-tmp\back.svg - Does it exist? true
Fatal error: Command failed: C:\AEM\AEM 6.1 full version\newinfracodebase\r16\sbg_aem\ui.apps\src\main\content\jcr_root\etc\designs\sbg\node_modules\grunt-grunticon\node_modules\grunticon-lib\node
odules\svg-to-png\node_modules\phantomjs-prebuilt\lib\phantom\bin\phantomjs.exe C:\AEM\AEM 6.1 full version\newinfracodebase\r16\sbg_aem\ui.apps\src\main\content\jcr_root\etc\designs\sbg\node_mo
les\grunt-grunticon\node_modules\grunticon-lib\node_modules\svg-to-png\lib\phantomscript.js
```

The error occurred during execution of the grunt-icon task. The grunt-icon task uses `phantomjs.exe` file to convert the svgs to pngs.
`phantomjs.exe` is pulled from the npm repo if you try to install the grunt-icon task.
 
The issue with `phantomjs.exe` is that the latest version of the exe (2.1) the mentioned error.
Replacing the `phantomjs.exe` to an older version resolves this issue.

You can locate the `phantomjs.exe` at following location under your node-modules –
`node-modules\sbg\node_modules\grunt-grunticon\node_modules\grunticon-lib\node_modules\svg-to-png\node_modules\phantomjs-prebuilt\lib\phantom\bin`
 
Replace the `phatomjs.exe` located in the above directory with an exe available here –
https://storage.googleapis.com/google-code-archive-downloads/v2/code.google.com/phantomjs/phantomjs-1.9.2-windows.zip

Following the above mentioned process will resolve the grunt issue.
